packages/docs/src/locales/en/en.json

```diff
@@ -758,7 +758,8 @@
       },
```

Comment:
On mouse down:

![image](https://user-images.githubusercontent.com/5394573/145555454-f7174d8a-6a7c-464f-8ca1-49f08c229fe1.png)

Normal:

![image](https://user-images.githubusercontent.com/5394573/145555544-15cf20a8-6a8c-4420-9be1-6d2da86a2d33.png)

You can see outline on click. That shouldn't happen.

Reply:
Fixed it